### DISCUSSION TITLE
Pagina Creditis

### DISCUSSION THREAD

**Discussion by gbvitrano** [opendatasicilia/opendatasicilia.it#53] [Idee/Proposte]
Una pagina  credits del gruppo che si è occupato in passato ed ora del sito web  la facciamo ?
[ esempio](https://opendatasicilia.it/credits/)
![Acquisisci schermata Web_13-12-2021_15308_opendatasicilia it](https://user-images.githubusercontent.com/20488693/145831083-0b111cfd-0c8f-42f6-b3d3-50310ad52724.jpeg)

**Comment by dennisangemi** [comment]
Bella idea @gbvitrano , bisognerebbe capire bene come metterla in relazione alla pagina _Chi siamo_ e ai profili degli/lle _authors_
